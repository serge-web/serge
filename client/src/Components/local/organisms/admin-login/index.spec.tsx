/* global it expect */
import React from 'react'
import renderer from 'react-test-renderer'

import AdminLogin from './index'

jest.mock('react-redux', () => {
  const reactRedux = jest.requireActual('react-redux')
  return {
    ...reactRedux, 
    useSelector: jest.fn(() => [])
  }
})

describe('AdminLogin component:', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<AdminLogin onSubmit={window.alert}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
