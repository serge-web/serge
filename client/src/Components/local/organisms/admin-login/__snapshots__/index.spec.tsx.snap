// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminLogin component: renders correctly 1`] = `
<div
  className="container"
>
  <span
    className="logo"
  >
    <img
      src=""
    />
    <p>
      Serge
    </p>
    <p>
      Admin Page
    </p>
  </span>
  <div
    className="password"
  >
    <section
      className="input-container disable-offset fullwidth"
    >
      <div
        className="MuiBox-root MuiBox-root-8 input-label"
      >
        Password
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        onKeyPress={[Function]}
      >
        <div
          className="MuiInputBase-root MuiFilledInput-root makeStyles-root-1 MuiInputBase-fullWidth MuiInputBase-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiFilledInput-input makeStyles-input-2"
            disabled={false}
            name="password"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="password"
            value="Nelson"
          />
        </div>
      </div>
    </section>
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Enter
    </span>
  </button>
</div>
`;
